nemo/collections/asr/modules/conv_asr.py

```diff
@@ -331,6 +331,57 @@ def num_classes(self):
         return self._num_classes
 
 
+class ConvASRDecoderRegression(NeuralModule, Exportable):
```

Comment:
This neural module can be refactored entirely inside `ConvASRDecoderClassification` by adding/removing the final activation function. I think having a shared base module between the two will greatly reduce this code duplication.